src/object_view.hpp

```diff
@@ -434,6 +320,63 @@ class object_view {
 
 static_assert(sizeof(object_view) == sizeof(void *));
 
+inline owned_object clone(object_view input)
```

Comment:
could be method on readable_object

Reply:
True, I'll move it in the next PR.

Reply:
Addressed in https://github.com/DataDog/libddwaf/pull/387